### PR TITLE
AT_16.02_003 Rename using dropdown menu-Rename with empty name field

### PR DESCRIPTION
**Pull request by Gayana-G**
https://trello.com/c/8QdQNqTg/571-at1602003-multibranch-pipeline-rename-pipeline-using-dropdown-menu-rename-with-empty-name-field-negative-scenario-2

### CODE REVIEW

File: cypress/e2e/multibranchPipelineRename.cy.js [hunk 1]
```diff
@@ -3,45 +3,56 @@
...
-        cy.get('a[href$="/newJob"]').click();
-        cy.get('.jenkins-input').as('nameInputField');
-        cy.get('@nameInputField').type(pipelineName.namePipeline);
-        cy.get('[class="label"]').contains('Multibranch Pipeline').click();
-        cy.get('button[type=submit]').should('be.enabled').click();
-        cy.url().should('include', `/${pipelineName.namePipeline}/configure`);
-        cy.get('button[name=Submit]').click();
-        cy.get('[class="model-link"]').contains('Dashboard').click();
-    })
+describe("US_16.02|<Multibranch Pipeline> Rename Multibranch Pipeline", () => {
```
Comment (Rom-T) [inline cypress/e2e/multibranchPipelineRename.cy.js:6]: Please name the describe as the name of your file.

Comment (Gayana-G) [reply cypress/e2e/multibranchPipelineRename.cy.js]: Thanks, Roman. 
Done

**Review by Rom-T — APPROVED** [review]
LGTM

**Review by rgirl0801 — COMMENTED** [review]
Approved

**Review by ViktoriyaED — APPROVED** [review]

**Pull request merged**